src/models/user.js

```diff
@@ -0,0 +1,18 @@
+import { DataTypes } from "sequelize";
+import { client } from "../utils/db.js";
+
+// defining columns in dataBase
+export const User = client.define('user', {
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: false
```

Comment:
The `unique` constraint for the `email` field is set to `false`. Typically, emails should be unique to prevent multiple accounts with the same email address. Consider setting `unique: true` to enforce this constraint.